### ISSUE TITLE
Support "apply from:" in Gradle

### ISSUE THREAD

**Issue by wtanaka** [dependabot/feedback#540]
It's possible to use "apply from" in gradle to include other .gradle files:

https://github.com/wtanaka/ciconfiggen/blob/cc8bfc43f7e9e4109310c9fb48af6ad83cdacf13/build.gradle#L2-L4

Based on what's shown in the web dashboard, it looks like dependabot might not be scanning dependencies in these included files:

https://app.dependabot.com/accounts/wtanaka/repos/178891901

**Comment by wax911** [comment]
Agreed since depndabot looks for a default configuration file and `build.gradle` an optional parameter to point to a custom location with a custom file name `*.gradle` would also be a good addition.

Since we can already define a `default directory` allowing the configuration to handle custom file would make a good addition.
